deps/rabbit_common/src/file_handle_cache.erl

```diff
@@ -1125,13 +1158,13 @@ handle_call({open, Pid, Requested, EldestUnusedSince}, From,
     case needs_reduce(State #fhc_state { open_count = Count + Requested }) of
         true  -> case ets:lookup(Clients, Pid) of
                      [#cstate { opened = 0 }] ->
-                         true = ets:update_element(
+                       safe_ets_update_element(
```

Comment:
Probably not important but based on the other changes I would expect `_ = safe_ets...` here.